include/nbl/builtin/hlsl/blur/box_blur.hlsl

```diff
@@ -0,0 +1,318 @@
+#pragma once
+
+#include <nbl/builtin/hlsl/blur/common.hlsl>
+
+#include <nbl/builtin/hlsl/glsl_compat/core.hlsl>
+#include <nbl/builtin/hlsl/workgroup/arithmetic.hlsl>
+#include <nbl/builtin/hlsl/workgroup/scratch_size.hlsl>
+
+
+// https://github.com/microsoft/DirectXShaderCompiler/issues/6144
+static nbl::hlsl::uint32_t3 /*nbl::hlsl::glsl::*/gl_WorkGroupSize() { return uint32_t3( WORKGROUP_SIZE, 1, 1 ); }
```

Comment:
you can forward declare this function and just let the user define it, instead of having a `WORKGROUP_SIZE` define we need to populate later